### PR TITLE
added --release flag to gsctl upgrade cluster

### PR DESCRIPTION
**Pull request by whites11**
This PR adds the reference of `--release` flag to `gsctl upgrade cluster`.

### CODE REVIEW

**Review by fiunchinho — APPROVED** [review]
LGTM

File: src/content/reference/gsctl/upgrade-cluster.md [hunk 1]
```diff
@@ -28,9 +28,18 @@ To prevent the interactive confirmation, you can use the `--force` flag. This wi
 gsctl upgrade cluster f01r4 --force
 ```
 
+If you want to upgrade to a specific version, you can use the `--release` flag:
+
+```nohighlight
+gsctl upgrade cluster f01r4 --release 13.0.0
+```
```
Comment (pipo02mix) [inline src/content/reference/gsctl/upgrade-cluster.md:35]: What about adding a notice (`__warning__:`) saying that skipping upgrades can cause unexpected impact?

Comment (whites11) [reply src/content/reference/gsctl/upgrade-cluster.md:35]: If skipping upgrades is problematic, we should block them using admission controllers.
We do that in azure

Comment (marians) [reply src/content/reference/gsctl/upgrade-cluster.md:35]: +1. Our admission controllers should prevent skipping majors. No need to raise discomfort here.

**Review by marians — APPROVED** [review]

File: src/content/reference/gsctl/upgrade-cluster.md [hunk 2]
```diff
@@ -28,9 +28,18 @@ To prevent the interactive confirmation, you can use the `--force` flag. This wi
...
+
+You can list available releases with `gsctl list releases`.
+
 ## Related
 
 - [Upgrade cluster general documentation](/reference/cluster-upgrades/)
 - [`gsctl create cluster`](/reference/gsctl/create-cluster/): Reference for creating a cluster
 - [`gsctl delete cluster`](/reference/gsctl/delete-cluster/): Reference for deleting a cluster
+- [`gsctl list releases`](/reference/gsctl/list-releases/): Reference for listing available releases
```
Comment (marians) [inline src/content/reference/gsctl/upgrade-cluster.md:44]: Thanks!

**Pull request merged**